pkg/get/get_test.go

```diff
@@ -474,3 +475,56 @@ func Test_DownloadInletsctl(t *testing.T) {
 		}
 	}
 }
+
+func Test_DownloadDoctl(t *testing.T) {
+	tools := MakeTools()
+	name := "doctl"
+
+	var tool *Tool
+	for _, target := range tools {
+		if name == target.Name {
+			tool = &target
+			break
+		}
+	}
+
+	type test struct {
+		os      string
+		arch    string
+		version string
+		url     string
+	}
+
+	const toolVersion = "1.46.0"
+	const urlPrefix = "https://github.com/digitalocean/doctl/releases/download"
+
+	tests := []test{
+		{os: "mingw64_nt-10.0-18362",
+			arch:    arch64bit,
+			version: toolVersion,
+			url:     urlPrefix + "/v" + toolVersion + "/doctl-1.46.0-windows-amd64.zip"},
+		{os: "linux",
+			arch:    arch64bit,
+			version: toolVersion,
+			url:     urlPrefix + "/v" + toolVersion + "/doctl-1.46.0-linux-amd64.tar.gz"},
+		{os: "darwin",
+			arch:    arch64bit,
+			version: toolVersion,
+			url:     urlPrefix + "/v" + toolVersion + "/doctl-1.46.0-darwin-amd64.tar.gz"},
```

Comment:
Can you switch to `fmt.Sprintf()` for these? 
```
const urlTemplate = "https://github.com/digitalocean/doctl/releases/download/v/%s/doctl-1.46.0-%s-amd64.tar.gz"
```
then use this in this way: 

```
url: fmt.Sprintf(urlTemplate, toolVersion, "windows")
```

Reply:
All set. Thanks!